Scripts/Services/Pet Training/AreaEffects.cs

```diff
@@ -411,17 +403,19 @@ public PoisonBreath()
 
         public override void DoEffect(BaseCreature creature, Mobile m)
         {
-            m.ApplyPoison(creature, creature.HitAreaPoison);
+            var def = PoisonBreathDefinition.GetDefinition(creature);
+
+            m.ApplyPoison(creature, def.Poison);
```

Comment:
The level of poison applied is proportional to the poisoning skill of the creature.

exm: 

```
int level = 1;
            double total = creature.Skills[SkillName.Poisoning].Value;

            if (total >= 100)
                level = 5;
            else if (total > 60)
                level = 4;
            else if (total > 40)
                level = 3;
            else if (total > 20)
                level = 2;
            else if (total > 0)
                level = 1;

m.ApplyPoison(creature, Poison.GetPoison(level));  
```

Reply:
Nothing was changed, the poison level/damage is just now consolidated in one script. This is area poison, not a direct poison attack.

Reply:
According to UOGuide: http://www.uoguide.com/Poison_Breath

Reply:
https://uo.com/wiki/ultima-online-wiki/skills/animal-taming/animal-training-abilities/

Reply:
I tested it at the OSl. I taught poison breath to the creature of Triton. He had 20 skills. Level 1 poison affected the player he fought.

Reply:
Ok! nevermind what I say lol.

Reply:
tasanar's account is inactive. I was going to share a screenshot.

Reply:
![1](https://user-images.githubusercontent.com/17765272/60917794-6a15fd00-a29a-11e9-8889-ea003932e7f3.PNG)
![2](https://user-images.githubusercontent.com/17765272/60917795-6a15fd00-a29a-11e9-9ad9-7728ca13a785.PNG)
![3](https://user-images.githubusercontent.com/17765272/60917796-6a15fd00-a29a-11e9-81a4-6fb9c64f3420.png)

Reply:
Also, Poison breath cannot be taught unless the creatures are taught poisoning

Reply:
![5](https://user-images.githubusercontent.com/17765272/60919032-60da5f80-a29d-11e9-82ce-f70e6897d379.png)